site/docs/voice/about.mdx

```diff
@@ -49,7 +57,37 @@ When calls or call events occur - Bandwidth will send a request (webhook) to you
 
 ### Webhooks
 
-The Voice API uses a number of different [webhooks](/docs/voice/webhooks) to facilitate call control. Certian webhooks expect BXML in the HTTP response to keep the call alive, while others are simply imformative.
+The Voice API uses a number of different [webhooks](/docs/voice/webhooks) to facilitate call control. Certain webhooks expect BXML in the HTTP response to keep the call alive, while others are simply informative. These are referred to as callbacks in the previous docs.
+
+### STIR/SHAKEN
+
+The Bandwidth API uses STIR/SHAKEN to help combat fraudulent calls and phone number spoofing. This info can be seen in the [`initiate`](/docs/voice/webhooks/initiate) webhook. For more information, look [here](https://www.bandwidth.com/regulations/stir-shaken/) for Bandwidth's explanation of what it is and why it's important.
+
+## Features and Services of Note
+
+### Text-to-Speech
+
+Use our [SpeakSentence](/docs/voice/bxml/speakSentence) verb to convert typed words to speech, powered by Amazon Polly. Seventeen voices, twenty-six languages. Use [SSML](https://www.w3.org/TR/2010/REC-speech-synthesis11-20100907/) to customize your audio to include pauses, emphasis, and much more.
+
+### Recording
+
+With the [Record](/docs/voice/bxml/record) verb, customers can save calls to play back later.
```

Comment:
I'm not personally a fan of referring to our customers as `customers` here, since these are external docs. I'd prefer `you`, but whatever we use, we should be consistent across our (voice API) docs.

Reply:
```suggestion
With the [Record](/docs/voice/bxml/record) verb, you can save calls to play back or analyze later.
```

Reply:
I was using 'you' before, but figured that was weirdly informal. Not sure about consistency, but I'll switch it.